dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/gateway/NettyServiceGateway.scala

```diff
@@ -224,18 +224,32 @@ class NettyServiceGateway(coordinatedShutdown: CoordinatedShutdown, config: Serv
       }
       for (header <- request.headers.asScala) {
         // Ignore connection headers
-        if (!header.getKey.equalsIgnoreCase(HttpHeaders.Names.CONNECTION)) {
+        if (!header.getKey.equalsIgnoreCase(HttpHeaderNames.CONNECTION.toString)) {
           newRequest.headers.add(header.getKey, header.getValue)
         }
       }
-      // todo - add Forwarded and other headers
+      // todo - add Forwarded and other headers.
```

Comment:
The original `todo` comment mentioned using `Forwarded` headers instead of `X-Forwarded-Host`. I preferred using `X-Forwarded-Host` over `Forwarded` because `Forwarded` is not supported in Akka-HTTP so we have the same implementation in both gateways.